Controllers/OrdersController.cs

```diff
@@ -0,0 +1,109 @@
+﻿using API.Models;
+using API.Models.DTO;
+using API.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore;
+using API.Models.DTO.Administrator;
+
+namespace API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class OrdersController : ApiControllerBase
+    {
+
+        private readonly UserManager<Employee> _userManager;
+
+        public OrdersController(ApiContext context, UserManager<Employee> userManager) :
+            base(context) {
+            _userManager = userManager; 
+        }
+
+        [HttpGet("orders")] //return a list of orders without products
+        public async Task<ActionResult<IEnumerable<GetOrdersDTO<ProductBalance>>>> GetOrders()
+        {
+            if (!IsValidApiRequest())
+            {
+                return ApiBadRequest("Invalid Headers!");
+            }
+
+            var orders = await Context.Order
+              .Select(o => new OrdersDTO<ProductBalance>()
+              {
+                  Id = o.Id,
+                  OrderTime = o.OrderTime,
+                  DeliveryTime = o.DeliveryTime,
+                  AddressFrom = o.AddressFrom,
+                  AddressTo = o.AddressTo,
+                  Status = o.Status,
+                  Employee = Context.Employees.Where(y => y.PersonalCode == o.Employee.PersonalCode).First().FirstName
```

Comment:
This should be moved to constructor so you could just do `new OrdersDTO(o)`. Also, do we really need to display employee name here? Maybe we could simplify this?

Reply:
Leave this for future.

Reply:
I think we could make this change with this MR and be done with it. Otherwise it might get forgotten.